File: ccc03s1.py
square = 1
while True:
    move = int(input())
    if move == 0:
        print("You Quit!")
        break
    square += move
    if square == 54:
        square = 19
    elif square == 90:
        square = 48
    elif square == 99:
        square = 77
    elif square == 9:
        square = 34
    elif square == 40:
        square = 64
    elif square == 67:
        square = 86

    elif square >= 100:
        print("You are now on square 100")
        print("You Win!")
        break

    
        
    print("You are now on square " + str(square))

File: ccc03s1.cpp
#include <iostream>
using namespace std;

int square = 1;
int main(void)
{ 
    while (true) {
        int move; 
        cin >> move;
        if (move == 0) {  
            cout << "You Quit!";
            break;
        }
        square = square + move;
        if (square == 54) { 
            square = 19;
        } else if (square == 90) { 
            square = 48;
        } else if (square == 99) { 
            square = 77;
        } else if (square == 9) { 
            square = 34;
        } else if (square == 40) {
            square = 64;
        } else if (square == 67) { 
            square = 86;
        } else if (square > 100) {
            square = square - move;
        } else if (square == 100) {
            cout << "You are now on square 100\n";
            cout << "You Win!";
            break;
        }
        
        cout << "You are now on square "; 
        cout << square;
        cout << "\n";
    }
} 
